src/components/overlays/TheCreateNewTacticOverlay.vue

```diff
@@ -126,18 +122,23 @@ export default class CreateNewTacticOverlay extends mixins(TacticWatcher) {
     imgName: HTMLHeadingElement;
   }
 
+  isDisabled (): boolean {
+    return !(this.$data.tactic.name && this.$data.tactic.map.icon)
+  }
+
   // Need to do more to this, but we dont have the collection stuff created yet so this is temporary.
   createTactic (): void {
-    if (this.$data.tactic.name && this.isAuth && this.jwt !== null) {
-      EventBus.$emit('loadCanvas')
+    const foundCollection = this.collections.find((collection: Collection) => collection.name === 'root')
```

Comment:
Should we not be able to specify which collection we want to create the tactic in?

Reply:
Initially there is only one collection, the root collection. As we go into MVP2 subcollections will be a thing so this will be changed to allow a user to create or select their own